content/courses/onchain-development/anchor-pdas.md

```diff
@@ -110,7 +116,7 @@ pub struct InitializePda<'info> {
         seeds = [b"example_seed", user.key().as_ref()],
         bump,
         payer = user,
-        space = 8 + 8
+        space = (DISCRIMINATOR as usize) + Accountype::INIT_SPACE
```

Comment:
We normally just make DISCRIMINATOR be a usize in the first place (than than a u8 and then coercing it). What do you think?

Reply:
DISCRIMINATOR is u8 in Anchor so I thought that making the constant also u8 might be a good idea. That sort of would be a good way to relate ig...
well we can make DISCRIMINATOR usize in the lesson, not a problem either.